cfy_manager/components/restservice/config/authorization.conf

```diff
@@ -925,6 +925,12 @@ permissions:
     - user
     - operations
     - viewer
+  widget_environmentButton:
+    - sys_admin
+    - manager
+    - user
+    - operations
+    - viewer
```

Comment:
I think we shouldn't have `viewer` here (similarly to `widget_deploymentButton`) as according to [Managing Roles](https://docs.cloudify.co/staging/dev/working_with/manager/roles-management/) it's not possible to create deployment with `viewer` role.

Reply:
Fair point, corrected.